src/app/core/session/auth/local/local-auth.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from "@angular/core";
+import { AuthUser } from "../auth-user";
+
+@Injectable({
+  providedIn: "root",
+})
+export class LocalAuthService {
+  private static LAST_LOGGED_IN_KEY = "LAST_USER";
+
+  login(): AuthUser {
+    return this.getStoredUser(LocalAuthService.LAST_LOGGED_IN_KEY);
```

Comment:
`getStoredUser()` doesn't seem to get reused and is itself only a short method. I think it might be more readable to inline it here.